frontend/views/containers/modals/CreateGroup.vue

```diff
@@ -46,8 +43,7 @@ modal-base-template
           @click='submit'
           :disabled='$v.form.$invalid'
           data-test='finishBtn'
-        )
-          i18n Create Group
+        ) Create Group
```

Comment:
It was fine before. Now it's no longer localized. Either revert, or replace line 40 with `i18n(tag='button').is-success`